### PR TITLE
[WIP] [match] Give output more structure

### PR DESCRIPTION
**Pull request by janpio**
### Checklist
- [ ] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [ ] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [ ] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### TODO

- [ ] Better output for reading foo, creating foo, installing foo
- [ ] handle https://github.com/fastlane/fastlane/pull/13943/commits/89da896d167f8c7ec33c05601ffe37a182e788d1

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Pull Request is Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: match/lib/match/runner.rb [hunk 1]
```diff
@@ -46,13 +46,20 @@ def run(params)
...
+      UI.message("       Download from Storage        ")
+      UI.message("====================================")
       storage.download
 
       # Init the encryption only after the `storage.download` was called to have the right working directory
       encryption = Encryption.for_storage_mode(params[:storage_mode], {
         git_url: params[:git_url],
         working_directory: storage.working_directory
       })
+      UI.message("====================================") if encryption
```
Comment (armcburney) [inline match/lib/match/runner.rb]: Could you please write this as:

```ruby
if encryption
  UI.message("====================================")
  UI.message("           Decrypt files            ")
  UI.message("====================================")
  encryption.decrypt_files
end
```
___

Also, do you think we could use the [`UI.header`](https://github.com/fastlane/fastlane/blob/f32b007ff45e648b37b6c9c2037ac481f36b7780/fastlane_core/lib/fastlane_core/ui/implementations/shell.rb#L86) function instead?

Comment (janpio) [reply match/lib/match/runner.rb]: Oh yes, very much. I just did this while creating #13943 but didn't want to just throw it away, so parked it here in this branch (and PR so I don't forget about it).

Comment (armcburney) [reply match/lib/match/runner.rb]: Ahh, gotcha - sounds good to me! :)

**Pull request closed without merging**